starwars/src/components/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './StarWars.css'
+
+function Card(props) {
+    const data = props.data
+    return (
+        <div style={{ color: "dark brown" }}>
+        <div className="card">
```

Comment:
Since both of these top two divs are more presentational and aren't being used for separating styles, you only need one of them.